src/vm/arm/ehhelpers.S

```diff
@@ -99,6 +99,7 @@ OFFSET_OF_FRAME=(4 + SIZEOF__GSCookie)
         NESTED_ENTRY CallEHFunclet, _TEXT, NoHandler
 
         PROLOG_PUSH  "{r4-r11, lr}"
+        PROLOG_STACK_SAVE r7
         alloc_stack  4
 
```

Comment:
@janvorli I reverted this change and changed code as you mentioned, but it looks like that unwind still fails.